content/en/containers/docker/apm.md

```diff
@@ -291,9 +291,40 @@ from ddtrace import tracer
 tracer.configure(hostname='172.17.0.1', port=8126)
 ```
 
+### Unix Domain Socket (UDS)
+To submit traces via socket, it should be mounted to the Agent container and your application container.
```

Comment:
Can you clarify what "it" is in the context of "being mounted to the Agent container and your application container"?

Reply:
"it" is socket. Otherwise it's not accessible by the agent and tracer.